File: src/ui/StageController.java
package ui;

import javafx.stage.Stage;

/**
 * Created by brian on 8/29/15.
 */
public class StageController {

    private Stage stage;

    public Stage getStage() {
        return stage;
    }

    public void setStage(Stage stage) {
        this.stage = stage;
        stage.show();
    }

    public void showStage() {
        this.stage.setWidth(1024);
        this.stage.setHeight(768);
        this.stage.setMaximized(true);
    }
}

File: src/ui/StageController.kt
package ui

import javafx.stage.Stage

/**
 * Created by brian on 8/29/15.
 */
class StageController {
    var stage: Stage? = null

    fun showStage() {
        if (this.stage != null) {
            this.stage!!.width = 1024.0
            this.stage!!.height = 768.0
            this.stage!!.show();
        }
    }
}
